cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/AbstractConnectOperator.java

```diff
@@ -593,6 +603,184 @@ private Future<List<Condition>> maybeRestartConnectorTask(Reconciliation reconci
         }
     }
 
+    /**
+     * Maybe list, alter or reset the connector offsets.
+     * The action is based on the {@code strimzi.io/connector-offsets} annotation:
+     *   * {@code list} writes the offsets to a ConfigMap
+     *   * {@code alter} alters the offsets based on a ConfigMap
+     *   * {@code reset} resets the offsets
+     */
+    @SuppressWarnings({ "rawtypes" })
+    /* test */ Future<List<Condition>> manageConnectorOffsets(Reconciliation reconciliation, String host, KafkaConnectApi apiClient, String connectorName, CustomResource resource, KafkaConnectorSpec connectorSpec, List<Condition> conditions) {
```

Comment:
For all these methods, I think adding more spacing would help the readability. E.g. empty line aftet the catch / before the switch ... in the next methof empoty line before and after the `if` condition etc.